modules/openapi/conf/conf.go

```diff
@@ -60,6 +60,9 @@ type Conf struct {
 
 	// Allow people who are not admin to create org
 	CreateOrgEnabled bool `default:"false" env:"CREATE_ORG_ENABLED"`
+
+	// DocVersion version of document
+	DocVersion string `default:"1.0" env:"DOC_VERSION"`
```

Comment:
I think we already have some way to get base version of erda, no need to add a new env.